pkgs/development/compilers/elm/default.nix

```diff
@@ -274,6 +274,19 @@ in lib.makeScope pkgs.newScope (self: with self; {
         }
       );
 
+      elm-land = nodePkgs."elm-land".overrideAttrs (
+        old: {
+          nativeBuildInputs = (old.nativeBuildInputs or []) ++ [ makeWrapper old.nodejs.pkgs.node-gyp-build ];
```

Comment:
I don't understand why this is necessary.

Reply:
Bah that's an artifact from copy/pasting the definition from `elm-spa` in the same file. I'll remove it. Thanks for catching it!